src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -4207,8 +4215,6 @@ Map {
       "size": Object {
         "args": Array [
           Array [
-            "SMALL",
-            "SMALLWIDE",
```

Comment:
To avoid a breaking change, we'll need to leave these in. If someone has a small barchart card configured today and upgrades, what would be the best thing to happen? render `null`, render 1 size up (medium, mediumthin)? I'm not sure, thoughts? In addition I think we can mark these sizes deprecated to be removed in the next major.

Reply:
I think the least invasive thing to do is to render 1 size up. `SMALL` -> `MEDIUM`, `SMALLWIDE` -> `MEDIUMWIDE`.

I've added the deprecation notice, but made it custom so that I could add the additional rules